src/nwb/NWBFile.hpp

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <cstdint>
+#include <map>
```

Comment:
```suggestion
#include <array> 
#include <string_array>
```

Do we need to add these includes?